src/components/SeedBank/StorageLocations.tsx

```diff
@@ -36,30 +52,122 @@ export default function SubLocations({ seedBankId }: SubLocationsProps): JSX.Ele
       }
     };
 
-    if (isEnabled('Storage locations')) {
-      fetchStorageLocations();
+    if (featureEnabled) {
+      if (seedBankId) {
+        fetchStorageLocations();
+      } else {
+        setStorageLocations(DEFAULT_STORAGE_LOCATIONS().map((name, index) => storageLocationWith(name, index)));
+      }
+    }
+  }, [seedBankId, activeLocale, featureEnabled]);
+
+  const getTopBarButtons = () => {
+    const topBarButtons: TopBarButton[] = [];
+    if (selectedRows.length) {
+      topBarButtons.push({
+        buttonType: 'destructive',
+        buttonText: strings.DELETE,
+        // we don't want to delete locations that have active accessions
+        disabled: !selectedRows.some((location) => !location.activeAccessions),
+        onButtonClick: () => deleteStorageLocations(selectedRows.filter((location) => !location.activeAccessions)),
```

Comment:
Should be show a message explaining why the active accessions are not deleted?

Reply:
Current plan (separate jira) is to disable the checkbox here and in the inventory list - we don't allow withdraw on batches with 0 ready quantity. Same treatment here.